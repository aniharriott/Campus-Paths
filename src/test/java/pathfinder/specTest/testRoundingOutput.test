# Test that findMinPath will return least cost path
# rounded properly

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 6.1234
AddEdge graph1 n2 n3 8.9999

FindPath graph1 n1 n3