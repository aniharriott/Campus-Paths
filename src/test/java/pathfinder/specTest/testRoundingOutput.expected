# Test that findMinPath will return least cost path
# rounded properly

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 6.1234 from n1 to n2 in graph1
added edge 8.9999 from n2 to n3 in graph1

path from n1 to n3:
n1 to n2 with weight 6.123
n2 to n3 with weight 9.000
total cost: 15.122